### ISSUE TITLE
Clarity of note on inband parameters

### ISSUE THREAD

**Issue by poolec** [cta-wave/CMAF-Byte-Stream#29]
NOTE 2 in section 3 reads as if it has been heavily edited and it's quite hard to follow.

Suggested rewording:

NOTE 2: In the ISO BMFF byte stream format, whereas codec configurations stored out-of-band are required to be supported, the user agent is only recommended to support codec configurations stored inband in the samples themselves. In the CMAF byte stream format specification, no such distinction is made and any sample entry that is permitted in a CMAF media profile is required to be supported for that media profile.

**Comment by haudiobe** [comment]
AHG 2020/11/19: Agreed.

**Comment by haudiobe** [comment]
Implemented: https://1drv.ms/w/s!AiNJEPgowJnWgqsz3EFQvnMD7BhN0w?e=42BwJJ
